content/account_management/rbac/log_management.md

```diff
@@ -0,0 +1,83 @@
+---
+title: Role Based Access Control
+kind: documentation
+aliases:
+  - /guides/rbac
+further_reading:
+- link: "account_management/rbac/role_api"
+  tag: "Documentation"
+  text: "Manage Roles and Permissions with the Role API"
+---
+
+<div class="alert alert-warning">
+This feature is currently in Private Beta. Ask your Sales representative or Customer Success Manager to have it enabled.
+</div>
+
+## Roles
+
+Roles categorize users and define what account permissions those users have, such as what data they can read or what account assets they can modify. By default, Datadog offers three roles, and you can create custom roles so you can define a better mapping between your users and their permissions.
+
+For example, within Log Management you can specify which users can read which log data, and which users can manage log-related account assets, like pipelines, indexes, archives, etc.
+
+By granting permissions to roles, any user who is associated with that role receives that permission. When users are associated with multiple roles, they receive all the permissions granted to each of their roles. The more roles a user is associated with, the more access they will have within a Datadog account.
+
+## Out of the Box Roles
+
+The following roles are available out of the box. By default, users will be associated with one of these three roles:
+
+* **Datadog Admin Role**: Has read and write permissions on all data and assets in a Datadog account.
+
+* **Datadog Standard Role**: Has read permissions on all data and write permissions on most assets in a Datadog account.
+
+* **Datadog Read Only Role**: Has read permissions on all data and write permissions on no assets in a Datadog account.
+
+Users who are associated with any of these roles will, by default, be able to query on all log data in a Datadog account. Users who are associated with the Datadog Admin or Standard Roles will, by default, be able to create or modify all log-related account assets. But the Datadog Standard and Read Only Roles can be modified to limit their permissions.
+
+## Custom Roles
+
+Today, custom roles are imported via SAML integrations from Identity Providers: Datadog can receive the user groups from your IdP and automatically generate roles to match those user groups. In this case, users who sign in via the IdP will automatically be associated with those roles and will have the permissions that are granted to those roles. 
+
+Alternatively, you can also use our "Role" API to create roles, and associate users with those roles.
```

Comment:
let's link to the role API here. also, let's call it the `Role API` and not the `"Role" API` because the latter has a kind of "unnecessary quotation marks" feeling